newroutes/auth.js

```diff
@@ -0,0 +1,99 @@
+const autoBind = require("auto-bind")
+const express = require("express")
+
+// Import middleware
+const { attachReadRouteHandlerWrapper } = require("@middleware/routeHandler")
+
+const { FRONTEND_URL } = process.env
+
+const AUTH_TOKEN_EXPIRY_MS = parseInt(
+  process.env.AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS,
+  10
+)
+const CSRF_TOKEN_EXPIRY_MS = 600000
+const CSRF_COOKIE_NAME = "isomer-csrf"
+const COOKIE_NAME = "isomercms"
```

Comment:
wdyt about shifting these to a `constants` file? (also ok with leaving it here if it's too much trouble)

Reply:
As discussed offline, we've opted to leave it in this file as it is only used for auth and its corresponding test file. https://github.com/isomerpages/isomercms-backend/pull/328/commits/596c552abad77de364dcc905ace2ccb51cac3666